plugins/usbdmx/UsbDmxPlugin.cpp

```diff
@@ -107,6 +108,16 @@ bool UsbDmxPlugin::SetDefaultPreferences() {
 
   return true;
 }
+
+void UsbDmxPlugin::ConflictsWith(
+  std::set<ola_plugin_id>* conflicting_plugins) const {
```

Comment:
This looks a bit C++11 to me, or at least it's unfamiliar to me, the Internet seems to agree. It looks like the ternary operator might be able to solve the same thing for C++98 compatibility.

Reply:
I don't see C++11 usage here; what statement are you referring to?

Reply:
Sorry, the const constructor.

Reply:
Did I introduce a const constructor? If I did, that's a mistake... But I looked through the pull request and I don't see a const constructor anywhere.

Reply:
This is one unless I'm mistaken: https://github.com/OpenLightingProject/ola/pull/1556/files#diff-4fde8cb25f836b0c2bc0d0dbe5046ef0R113

Reply:
> This is one unless I'm mistaken: https://github.com/OpenLightingProject/ola/pull/1556/files#diff-4fde8cb25f836b0c2bc0d0dbe5046ef0R113

This is not a constructor, this overrides the `virtual void Plugin::ConflictsWith(std::set<ola_plugin_id>*) const {}`, which is already const in its parent, so has to be const. I don't understand what you want to change; could you maybe do that yourself?

Reply:
Really sorry @aroffringa , I've just realised what I'd misread, can you indent the std::set four from the void so it doesn't align with the line below (which can stay where it is). I was carelessly misreading it as:
```
void UsbDmxPlugin::ConflictsWith() {
  std::set<ola_plugin_id>* conflicting_plugins) const {
    if (EuroliteProFactory::IsEuroliteMk2Enabled(m_preferences)) {
      conflicting_plugins->insert(OLA_PLUGIN_USBPRO);
      conflicting_plugins->insert(OLA_PLUGIN_STAGEPROFI);
      conflicting_plugins->insert(OLA_PLUGIN_FTDIDMX);
    }
  }
}
```

I was rather confused why Travis wasn't complaining given what I'd read.

Reply:
Sure, fixed.